### DISCUSSION TITLE
Flashback when accessed via categories

### DISCUSSION THREAD

**Discussion by truman0102** [alshedivat/al-folio#2015] [Q&A]
My post is redirected to a PDF file. When I click directly from the blog page I can redirect to the PDF file, however, when I access it through the category it does not redirect to the PDF file but flashes back to the blog page, why is that?

**Comment by george-gca** [comment]
Are you using the latest version of our code? This seems to be working just fine, as you can see on [our demo site](https://alshedivat.github.io/al-folio/blog/2022/) by clicking on `a post with redirect`.

**Reply by truman0102** [reply]
Updates are coming in too fast!
